.github/workflows/test.yml

```
@@ -14,13 +14,13 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [10.x, 12.x, 14.x]
+        node-version: [14.x, 18.x, 20.x]
     
     env:
       CI: true
 
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
     - name: Use Node ${{ matrix.node-version }}
       uses: actions/setup-node@v1
```

```suggestion
      uses: actions/setup-node@v3
```

While you're at it, might as well upgrade this action, since v1 is using node 12 to run scripts.

---

good catch